ModelBundle/MongoTrait/Statusable.php

```diff
@@ -1,44 +0,0 @@
-<?php
-
-namespace OpenOrchestra\ModelBundle\MongoTrait;
-
-use OpenOrchestra\ModelBundle\Document\EmbedStatus;
```

Comment:
There is a relation into the modelBundle, this way this modification cannot be done